dev/AnimatedVisualPlayer/AnimatedVisualPlayer.idl

```diff
@@ -0,0 +1,83 @@
+﻿[WUXC_VERSION_INTERNAL]
```

Comment:
I think these should be WUXC_VERSION_MUXONLY instead of "INTERNAL", since "INTERNAL" will tag them as experimental. But there was still an open discussion about some of these interfaces being not ship-ready. Did that close? Or will you move these to public later?

Reply:
The PR is waiting signoff in the internal repo? Let's just move all the work to GitHub and abandon the other PRs/branches.

Reply:
Okay.